resources/views/site/teamspeak.blade.php

```diff
@@ -36,6 +36,25 @@
             </ul>
         </div>
         <div class="col-sm-2">
+        </div>
+
+            </div> 
+<hr>
+    <center><h2>Link your TeamSpeak3 PC Client UUID to your ZTL Controller Profile to assign your roles:</h2></center>
+    <div class="row">
+        <div class="col-sm-2">
+        </div>
+        <div class="col-sm-8">
+            <ul>
+                <li>If connected to the ZTL TeamSpeak3 server, disconnect</li>
+				<li>In your TeamSpeak 3 menu go to Tools -> Identities (Ctrl-I)</li>
+                <li>Highlight & right click to copy your Unique ID to your clipboard</li>
+                <li>Proceed to your <a href="https://www.ztlartcc.org/dashboard/controllers/profile">ZTL Controller Dashboard</a> and scroll to the bottom to find your TS3 UID field</li>
+                <li>Paste your TeamSpeak3 UUID into the text box and click the Save button</li>
+                <li>Log into the ZTL TeamSpeak 3 server and your roles should not auto assign. If they fail to assign, log back out and then in. If the roles fail to assign, please reach out to a ZTL staff member.</li>
```

Comment:
Typo: "your roles should not auto assign" should be "your roles should auto assign"

Reply:
Amended